errors.go

```diff
@@ -0,0 +1,92 @@
+package webrtc
```

Comment:
This is a great pattern, I do a bad job of thinking of `errors` as values.

Is this something we should do in every package? Every time you open a package we will just have an `errors.go`

This way when we return an error people can intelligently act on them, really nice.

Reply:
It seems golint does have a prefference:
> errors.go:10:2: error var ConnectionClosedError should have name of the form ErrFoo
> errors.go:24:2: error var NoConfigError should have name of the form ErrFoo
> errors.go:38:2: error var CertificateExpiredError should have name of the form ErrFoo
> errors.go:39:2: error var NoTurnCredError should have name of the form ErrFoo
> errors.go:40:2: error var TurnCredError should have name of the form ErrFoo
> errors.go:41:2: error var ExistingTrackError should have name of the form ErrFoo
> errors.go:67:2: error var ModPeerIdentityError should have name of the form ErrFoo
> errors.go:68:2: error var ModCertificatesError should have name of the form ErrFoo
> errors.go:69:2: error var ModRtcpMuxPolicyError should have name of the form ErrFoo
> errors.go:70:2: error var ModIceCandidatePoolSizeError should have name of the form ErrFoo

Yikes!